### DISCUSSION TITLE
Single quoted vs double quoted string

### DISCUSSION THREAD

**Discussion by jeffmaury** [openapi-ts/openapi-typescript#1861] [Q&A]
Our project enforce single quoted strings. Is there a way to have the generated code use single quoted strings ?

**Comment by drwpow** [comment]
It’s generally bad practice to format or lint autogenerated code. The recommendation is to just ignore any generated files from any lint rules.

Previous versions of this project shipped with Prettier, but formatting is surprisingly heavy, and slow, and slows down everyone’s experience even if they’re not using it. Plus, better options are available such as Biome anyway, so it doesn’t make sense to confuse TS generation with linting & formatting (especially when TypeScript itself doesn’t have a formatter).

If you do require linting on autogenerated files, then just run your preferred formatter on this file after it’s generated. But in many cases this just adds slowness to projects.
